### PR TITLE
Kim's first commit

### PR DESCRIPTION
**Pull request by KingKimPFG**
Updating super secret code.  @gartee-john-PFG

### CODE REVIEW

File: King.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Class line 1
+testing line 2
+again with the testing
```
Comment (heathermartinPFG) [inline King.txt:3]: great work.

Comment (HooverJoePFG) [reply King.txt:3]: great work

**Review by HooverJoePFG — APPROVED** [review]
nice

File: King.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Class line 1
+testing line 2
```
Comment (SmithJasonPFG) [inline King.txt:2]: Good enough, I guess...

**Review by SmithJasonPFG — APPROVED** [review]

**Pull request merged**